client-v2/src/shared/components/input/ButtonCircularCaret.tsx

```diff
@@ -6,34 +6,45 @@ import ButtonCircular from './ButtonCircular';
 
 export const ButtonCircularWithTransition = ButtonCircular.extend<{
   highlight?: boolean;
+  small?: boolean;
 }>`
```

Comment:
I wanted to try to reuse the size types `'s' | 'm' | 'l'` from the very original ButtonDefault component but wasn't able to get it working. Open to suggestions.

Reply:
The problem here is that `ButtonCircular` overrides `height`, `width` and `padding` that are the things modified by the `size` prop in `ButtonDefault`. You could modify the implementation of `ButtonCircular` (if you wanted) to set the _props_ for `ButtonDefault` to achieve the required styles rather than just overriding their styles. It's up to you though.

Reply:
I think I'm going to leave as is for now because the caret image also needs to be resized and would rather leave the logic in one component for now. I think there is probably a separate PR here re-looking at making this component more resusable alongside other use cases, like the hover buttons and the large circular button on the Fronts list overlay...